components/constellation/pipeline.rs

```diff
@@ -545,6 +530,7 @@ impl UnprivilegedPipelineContent {
             self.script_chan.clone(),
             self.load_data.url.clone(),
         );
+        let (content_process_shutdown_chan, content_process_shutdown_port) = unbounded();
```

Comment:
Should this be unbounded or a one-shot channel?

Reply:
I'm not aware of crossbeam having a dedicated one-shot variant.

Reply:
I meant `bounded(1)`.

Reply:
for what reason?

Reply:
Possible space efficiency, but it probably doesn't matter that much.

Reply:
I see what you mean, counterintuitively, the [unbounded variant](https://github.com/crossbeam-rs/crossbeam/blob/7e2e2a67e8231c93a2dcaf95a15406babae31ccf/crossbeam-channel/src/flavors/list.rs#L148) appears to require less bookkeeping than the [bounded variant](https://github.com/crossbeam-rs/crossbeam/blob/7e2e2a67e8231c93a2dcaf95a15406babae31ccf/crossbeam-channel/src/flavors/array.rs#L60).